doc/lightning-pay.7.txt

```diff
@@ -30,6 +30,9 @@ The response will occur when the payment fails or succeeds.  Once a
 payment has succeeded, calls to *pay* with the same 'bolt11' will
 succeed immediately.
 
+If the command times out while an outgoing payment attempt is ongoing,
+you may call *pay* again to continue monitoring the payment attempt.
+
```

Comment:
timeout default value needs to be documented.  I'd probably choose 30 rather than 20: internet is pretty slow in Australia :)